tests/perception/high_level_semantics_situation_to_developmental_primitive_perception_test.py

```diff
@@ -40,3 +40,32 @@ def test_person_and_ball():
     assert person_and_ball_perception.frames[0].relations
     print(person_and_ball_perception.frames[0].property_assertions)
     assert len(person_and_ball_perception.frames[0].property_assertions) == 2
+
+
+def test_person_and_ball_color():
+    person = SituationObject(PERSON)
+    ball = SituationObject(BALL, properties=[RED])
+
+    person_and_ball_situation = HighLevelSemanticsSituation(
+        ontology=GAILA_PHASE_1_ONTOLOGY, objects=[person, ball]
+    )
+
+    person_and_ball_perception = _PERCEPTION_GENERATOR.generate_perception(
+        person_and_ball_situation, chooser=RandomChooser.for_seed(0)
+    )
+
+    perceived_objects = person_and_ball_perception.frames[0].perceived_objects
+    print(perceived_objects)
+    assert len(person_and_ball_perception.frames) == 1
+    print(person_and_ball_perception.frames[0].property_assertions)
+    assert len(person_and_ball_perception.frames[0].property_assertions) == 3
+
+    property_handles = set(
+        repr(assertion)
```

Comment:
avoid directly testing the `repr`; instead define your own stringification in-place